packages/web/src/components/notification/Notification/FavoritedOfRepostNotification.tsx

```diff
@@ -0,0 +1,102 @@
+import { MouseEventHandler, useCallback } from 'react'
```

Comment:
nit - can we rename this file to `FavoriteOfRepostNotification` to match the mobile file name?